source/binary.cpp

```diff
@@ -281,13 +292,25 @@ spv_result_t Parser::parseInstruction() {
   // The zero values for all members except for opcode are the
   // correct initial values.
   spv_parsed_instruction_t inst = {};
-  inst.offset = _.word_index;
+  const size_t inst_offset = _.word_index;
```

Comment:
_.word_index is modified later in this method and by calls to parseOperand.  So we need to save it early.
But yes, I can move it down.